test/shared/speechRecognitionTests.jsx

```diff
@@ -42,6 +43,16 @@ const SpeechRecognitionTests = ({
       });
     });
 
+    describe('when the user blocks permission', () => {
+      beforeAll(() => {
+        MockRecognition.start = jest.fn(() => { throw new Error('not allowed'); });
+      });
+      
+      it.only('calls the onError function', () => {
```

Comment:
Oops this was a typo in the example I sent. `.only` is just for debugging as it skips the other tests in the file

Reply:
fixed!